### ISSUE TITLE
Make recruitment table responsive

### ISSUE THREAD

**Issue by svenanderzen** [armada-ths/ais#73]
Doesn't look good as it is for the moment.

**Comment by purmonen** [comment]
I agree it doesn't look good. However, the only way I can think of to make the table responsive is to remove some columns if the screen size is small. But removing information makes it less usable which I think is far worse than it not being responsive in this case since all information in the table is important for the recruitment task.

**Comment by svenanderzen** [comment]
Oh, I thought more in the way of wrapping the table in a bootstrap `<div class="table-responsive">`. That way well be able to move left and right in the table on mobile devices :) It's a quick and good fix that I'll do as soon as a get time :)

I made the tables like that on the branch `homepage-rebased` anyway. 

You can see and example here http://www.w3schools.com/bootstrap/tryit.asp?filename=trybs_ref_table-responsive&stacked=h

**Comment by purmonen** [comment]
I didn't know about that class. If it makes it better than go for it! I think there's way to much information on each row nevertheless to solve it by just making each column slightly smaller or making the font size smaller.

**Comment by svenanderzen** [comment]
Yeah, me neither actually I just found it when doing the homepage so just thought I would add it to a todo list by assigning myself so I didn't forget about it on the journey home ;)

I have to apologize as well, I should have written more info in the issue description and not just have dropped that line, but I was on my way to the subway and a bit short on time. I thought I would fix it right away when I got home but it fell into forgotten land... It was absolutely not meant as a blame towards you or anyone else :)

On every other note I totally with you agree on everything. By wrapping it in a `table-responsive` we'll still have the same column layout and sizing as I've understood it. The only difference will be that it will be wrapped in a separate, movable div instead of directly on to the viewport, thereby not moving outside of the viewport :)

*Note* I haven't got a DB with recruitment set up on my local machine yet so I haven't tried the change. I'll do that as soon as possible, but if someone feels like giving it a quick look I would be more that happy :) <3
